expect_test.go

```diff
@@ -84,18 +125,24 @@ func TestExpect(t *testing.T) {
 	if err != nil {
 		t.Errorf("Expected no error but got '%s'", err)
 	}
+	// close the pts so we can expect EOF
+	testCloser(t, c.Tty())
```

Comment:
If we call `c.Close()` first then there is a race condition where the goroutine will be trying to flush the response via `runeWriter.Flush()` or waiting for an EOF, but the ptm is closed causing `os.PathError`.  Closing the `c.Tty()` will correctly trigger the EOF for the goroutine, which will allow it to exit properly without errors.